web/relay/components/Repositories/RepoDependencies.tsx

```diff
@@ -0,0 +1,45 @@
+import { RepoMeta } from 'valist/dist/types';
+
+interface RepoDependenciesProps {
+  orgName: string,
+  repoName: string,
+  repoMeta: RepoMeta,
+  releaseMeta: any,
+}
+
+export default function RepoDependencies(props: RepoDependenciesProps): JSX.Element {
+  const { repoMeta, releaseMeta } = props;
+  let currentVersion;
+  let dependencies;
+  let devDependencies;
+
+  if (repoMeta.projectType === 'npm' && releaseMeta) {
+    currentVersion = releaseMeta.versions[Object.keys(releaseMeta.versions)[0]];
```

Comment:
We'll need to make this equal to the `releases[0].tag` to support other repo types, but this is awesome!